perun-base/src/main/java/cz/metacentrum/perun/core/api/AttributeDefinition.java

```diff
@@ -176,39 +170,32 @@ public int hashCode() {
 
 	@Override
 	public boolean equals(Object obj) {
-		if(obj == null) return false;
+		if (obj == null) return false;
 
 		if (!(obj instanceof AttributeDefinition)) return false;
 
 		final AttributeDefinition other = (AttributeDefinition) obj;
 
-		if (this.getId() != other.getId()) return false;
-		if(this.friendlyName == null ? other.friendlyName != null : !this.friendlyName.equals(other.friendlyName)) return false;
-
-		return true;
+		return this.getId() == other.getId() && (this.friendlyName == null ? other.friendlyName == null : this.friendlyName.equals(other.friendlyName));
 	}
 
 	@Override
 	public String serializeToString() {
-		StringBuilder str = new StringBuilder();
-
-		return str.append(this.getClass().getSimpleName()).append(":[").append(
-				"id=<").append(getId()).append(">").append(
-				", friendlyName=<").append(getFriendlyName() == null ? "\\0" : BeansUtils.createEscaping(getFriendlyName())).append(">").append(
-				", namespace=<").append(getNamespace() == null ? "\\0" : BeansUtils.createEscaping(getNamespace())).append(">").append(
-				", type=<").append(getType() == null ? "\\0" : BeansUtils.createEscaping(getType())).append(">").append(
-				']').toString();
+		return this.getClass().getSimpleName() + ":[" +
```

Comment:
Any reason for changing usage of ``StringBuilder`` to string concatenation ? I believe we once changed all beans ``toString()`` and ``serializeToString()`` methods to use ``StringBuilder`` , since it should be quicker or less memory consuming.